### ISSUE TITLE
Coveralls breaking again

### ISSUE THREAD

**Issue by JustinAiken** [coverallsapp/orb#45]
All CircleCI jobs went from passing, to failing: 

```
sha256sum: release/coveralls-linux.tar.gz: No such file or directory
release/coveralls-linux.tar.gz: FAILED open or read
sha256sum: WARNING: 1 listed file could not be read

Exited with code exit status 1
```

**Comment by afinetooth** [comment]
@JustinAiken thanks for reporting. Same as this issue: https://github.com/coverallsapp/coverage-reporter/issues/165

Working on it now. Will update you here.

**Comment by afinetooth** [comment]
Suggestion in meantime:

set [fail-on-error input option](https://circleci.com/developer/orbs/orb/coveralls/coveralls#:~:text=boolean-,fail_on_error,-Whether%20to%20fail) to `false` for now:

```
fail-on-error: false
```

**Comment by JustinAiken** [comment]
We do have fail_on_error set to false, it still failed 😞

**Comment by afinetooth** [comment]
@JustinAiken hmm. Thanks.

**Comment by afinetooth** [comment]
Suggested workarounds for users of different extensions [here](https://github.com/coverallsapp/coverage-reporter/issues/165#issuecomment-2389349124).

**Comment by afinetooth** [comment]
@JustinAiken We believe we've patched this for now by updating the coveralls-checksums.txt file.

Can you please try again to see if it works for you now?

**Comment by afinetooth** [comment]
@JustinAiken we have reports from users of the [github-action](https://github.com/marketplace/actions/coveralls-github-action) and [coverage-reporter](https://github.com/coverallsapp/coverage-reporter) that our [patch](https://github.com/coverallsapp/coverage-reporter/issues/165#issuecomment-2389408065) has fixed the issue for them. 

I would love to make sure the same is the case for you with the [orb](https://circleci.com/developer/orbs/orb/coveralls/coveralls) before we move on to the root cause (and the `fail_on_error` thing).

**Comment by afinetooth** [comment]
Great, thanks @badmintonkid!

**Comment by afinetooth** [comment]
Tested internally. Closing for now.

@JustinAiken please let us know if you're still having any issue with the orb using coverage-reporter `v0.6.15`.

Or reach out directly to support@coveralls.io since you're part of a paid subscription. Message for your subscription's users will route to me.)

**Comment by afinetooth** [comment]
@JustinAiken re:

> We do have fail_on_error set to false, it still failed 😞
> 
> EDIT: Relevant snippet of our config:
> 
> ```yaml
>       - coveralls/upload:
>           flag_name: rspec_unit_tests
>           parallel: true
>           fail_on_error: false
> ```

We just released [`v2.2.2` of the coveralls orb](https://circleci.com/developer/orbs/orb/coveralls/coveralls), which extends the behavior of the `fail_on_error` option to **setup tasks**, like downloading, verifying and finding the coveralls binary.

Previously, `fail_on_error` only applied to failures of the **coveralls commands**, like `coveralls report` and `coveralls done`.

More [here](https://github.com/coverallsapp/orb/issues/40#issuecomment-2400895164).
